services/madoc-ts/src/frontend/site/pages/tasks/review-listing/ReviewNagivation.tsx

```diff
@@ -131,7 +137,7 @@ export const ReviewNavigation: React.FC<{
             if (handleNavigation) {
               e.preventDefault();
               if (pages[pg].tasks) {
-                handleNavigation(pages[pg].tasks[idx + 1].id, pg + 1);
+                handleNavigation(pages[pg].tasks[idx + 1].id, pg + 1, idx + 2 > pages[pg].tasks.length - 1);
```

Comment:
`tasks[idx + 1]` might be undefined if it's the last time in a page

Reply:
It checks first to see if its getting the next item in the current page

` idx < pages[pg].tasks.length - 1 ? `

then it would go to `getNextPage()` where it gets the first item in the next page